pybossa/api/auditlog.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf8 -*-
+# This file is part of PYBOSSA.
+#
+# Copyright (C) 2015 Scifabric LTD.
```

Comment:
this can be updated to latest period

Reply:
What do you mean by the latest period? 2019?

Reply:
yes